SwiftWisdom/Core/StandardLibrary/Numbers/CountableClosedRange+Extensions.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  CountableClosedRange+Extensions.swift
+//  SwiftWisdom
+//
+//  Created by Andrew Dolce on 11/21/16.
+//  Copyright © 2016 Intrepid. All rights reserved.
+//
+
+import Foundation
+
+public extension CountableClosedRange where Bound: SignedInteger {
```

Comment:
Well, if there can indeed be different strides, then this is kind of worrying...we then need to make sure that the random output falls on the stride boundary, right? am I thinking about this right? 0...4 with stride 2 only had 0,2,4 as valid values.